lms/djangoapps/grades/new/course_grade.py

```diff
@@ -317,7 +317,7 @@ def create(self, course, read_only=True):
         if not self._user_has_access_to_course(course_structure):
             raise PermissionDenied("User does not have access to this course")
         return (
-            self._get_saved_grade(course, course_structure) or
+            self.get_saved_grade(course, course_structure) or
```

Comment:
If we are promoting this to a public method, I think its name should be in line with `create` and `update`.